### ISSUE TITLE
Problem with backspace and alphanumeric caractere

### ISSUE THREAD

**Issue by t0sa** [JiHong88/suneditor#868]
I use SunEditor with ReactJS and I install SunEditor with `npm install sundeditor --save`

SunEditor version : 2.41.3.

**Describe the bug**
In SunEditor, when I write a sentence like "Hello world, I use SunEditor" and I press [Backspace] key, the full sentence is deleted and not just the "r".

But if I start the sentence with a special character like this "@ Hello [...] SunEditor" and  I press [Backspace] key so the last character is deleted and not the full sentence

**Expected behavior**
When [Backspace] key is pressed, only one character is deleted.

![Hnet-image (1)](https://user-images.githubusercontent.com/60540517/132206070-e3fa63f5-8fc2-40cb-9c74-038b1203727a.gif)

**Desktop (please complete the following information):**
 - OS: [Big Sur]
 - Browser: [chrome]

**Comment by JiHong88** [comment]
@vphu This issue is not reproduced.
Please explain in more detail.

**Comment by vphu** [comment]
Ok.

First, I create a component like this :

```
import React from "react";

import SunEditor from "suneditor-react";
import 'suneditor/dist/css/suneditor.min.css';

export default class Editor extends React.Component{

  render(){
     <div>
        <SunEditor />
     </div>
  }

}
```

In the React component that includes SunEditor, I write some sentence.
When I want to delete a character with the backspace key. It is the whole sentence which is deleted and not the desired character (as in the gif above).

But if the sentence starts with a character other than a letter or a number then I can delete my sentence character by character.

So, I tried with your code sample : 

```
import React from 'react';
import SunEditor from 'suneditor-react';
import 'suneditor/dist/css/suneditor.min.css'; 

const MyComponent = props => {
  return (
    <div>
      <p> My Other Contents </p>
      <SunEditor />
    </div>
  );
};
export default MyComponent;
```

I have a same problem, I can't delete my sentence character by character. So it's a problem because if I want delete only one word, it's not possible and I have to rewrite my whole sentence.

I have this problem on Google Chrome [Version 93.0.4577.63], MacOs BigSur and High Sierra.
I tried on Safari but it's worse, I can't write a sentence longer than one character, the new character replaces the previous one.

Do you have enough details ?

**Comment by JiHong88** [comment]
Still can't reproduce the issue. :(
Does also have the same issue this [link](http://suneditor.com/sample/index.html)?

**Comment by vphu** [comment]
No I don't have this issue with this link.

I don't understand :/

**Comment by JiHong88** [comment]
@vphu No errors occur?
Do other functions work normally?

**Comment by vphu** [comment]
@JiHong88 yes, with your link, no error and other functions work normally.
I will try to reinstall SunEditor and try in a new project.

Thx
